Lib9c/BlockHelper.cs

```diff
@@ -27,7 +25,8 @@ public static class BlockHelper
             IImmutableSet<Address> activatedAccounts = null,
             bool isActivateAdminAddress = false,
             IEnumerable<string> credits = null,
-            int maximumTransactions = 100
+            int maximumTransactions = 100,
+            PrivateKey privateKey = null
```

Comment:
Shouldn't this parameter be required and get rid of default value?

Reply:
`privateKey` for `BlockHelper.MineGenesisBlock()` is still optional. I turned only `--private-key` option for `9c-tools genesis` because `BlockHelper.MineGenesisBlock()` can be used in other situations (e.g. unittest)...